notebooks/Binary Classification on COMPAS dataset.ipynb

```diff
@@ -199,10 +208,11 @@
     "from fairlearn.postprocessing import ThresholdOptimizer\n",
     "from copy import deepcopy\n",
     "\n",
-    "unconstrained_predictor_wrapper = LogisticRegressionAsRegression(unconstrained_predictor)\n",
+    "estimator_wrapper = LogisticRegressionAsRegression(estimator).fit(X_train, y_train)\n",
     "postprocessed_predictor_EO = ThresholdOptimizer(\n",
-    "    unconstrained_predictor=unconstrained_predictor_wrapper,\n",
-    "    constraints=\"equalized_odds\")\n",
+    "    estimator=estimator_wrapper,\n",
+    "    constraints=\"equalized_odds\",\n",
+    "    refit=False)\n",
     "\n",
```

Comment:
prefit=True

Reply:
Did you mean to check in an updated notebook or leave this for a future PR? There's another place in this notebook, where we should use `prefit=True`.

Reply:
You need to fix the notebook. There are (gated) tests that won't succeed unless the notebooks are up to date.